app/main.py

```diff
@@ -1,2 +1,97 @@
+from typing import Any, Hashable
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
```

Comment:
also in the initializer it is necessary to use the load factor and capacity for this job